packages/functional-tests/pages/settings/index.ts

```diff
@@ -65,6 +65,35 @@ export class SettingsPage extends SettingsLayout {
     ]);
   }
 
+  async disconnectSync(creds) {
+    await this.goto();
+    const services = await this.connectedServices.services();
+    const sync = services.find((s) => s.name.includes(' on '));
+    
+    if (sync) {
+      console.log('disconnecting sycn');
```

Comment:
I'm referencing this PR to help me work on my own related ticket and just noticed: ~sycn~ sync